public/data/web3community.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "Web3 Community",
+  "bio": "An Open Source Community that focuses on decentralized applications, web 3, and blockchain technologies 🚀",
```

Comment:
```suggestion
  "bio": "An Open Source Community that focuses on decentralized applications, Web 3.0, and Blockchain technologies 🚀",
```

Reply:
It should lowercase as per grammarly.

Reply:
> It should lowercase as per grammarly.

Not for my Grammarly settings, though 👀 let's keep it as-is anyways.
But as you can see on the website we're using `Web 3.0`.

![image](https://user-images.githubusercontent.com/51391473/143382305-eb95b59d-3a31-413b-ade4-3f9aa3050048.png)

Reply:
As you can see in the github org bio, we are using web3 🙂